app/assets/stylesheets/common.scss

```diff
@@ -750,6 +750,11 @@ tr.turn:hover {
 
   &.new-user-main {
     background-image: image-url("sign-up-illustration.png");
+    background-position-x: 100px;
+  }
+
+  &.new-user-blocked {
+    background-image: image-url("sign-up-illustration.png");
```

Comment:
I started off asking if this also needed the position attribute but found that led me into the fact that the whole `users/blocked` view probably needs to be updated to match the new login/signup layout.

I've added an ACL to block the `blocked.example.com` domain on the test site so you can see what I mean by trying to signup with a `user@blocked.example.com` email.

Reply:
I can definitely remove the arm from this page.

But other than that, the page does not look as finished.

Reply:
The overlapping arm is actually an existing bug, as is the bizarrely massive font size. What I think is new is the relative width of that screen compared to the login screen which in turn means the header image is differently positioned.